server/src/main/java/org/tctalent/server/service/db/impl/CandidateServiceImpl.java

```diff
@@ -3035,4 +3035,41 @@ public Page<Candidate> fetchCandidatesWithChat(
         }
     }
 
+    @Override
+    public void reassignSavedListCandidates(SavedList savedList, int partnerId) {
```

Comment:
I might annotate this method as @Transactional in case of runtime exceptions; it will roll back.

Reply:
👍 Ah great, wasn't familiar with that dimension of the @Transactional annotation. Have read up and makes sense, will add.